Fix `test_fim/test_files/test_skip/test_skip.py`

|Related Issue|
|--|
|#1516 #1539|

```
Module: File Integrity Monitoring
Target: Manager and Agent   
OS: Linux
```

|  Type | Description  |
|--|--|
|Test Path | test_fim/test_files/test_skip/test_skip.py
| Consistent | Yes
| Test Execution | 3/3 executions failed
| Cases Failed | [get_configuration3-/nfs-mount-point-tags_to_apply0]
| Summary | `test_skip.py` is not restarting the nfs server correctly
| Error message |  AssertionError: No events should be detected.


In order to finish this issue the following tasks should be fulfilled:
- [x] Fails research.
- [x] Fix the test: In CentOS 8, the script that configures the nfs server must use `systemctl restart nfs-server`.
- [x] Full Green/Yellow in `test_fim/test_files/test_skip/test_skip.py`. 
- [x] Documentation of any important change done for these tests or the used tools.

--- a/tests/integration/test_fim/test_files/test_skip/data/configure_nfs_rpm.sh
+++ b/tests/integration/test_fim/test_files/test_skip/data/configure_nfs_rpm.sh
@@ -4,10 +4,20 @@ rpm -qa | grep -E "iproute.*" || yum -y install iproute
 ip=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
 # Check if nfs-utils is installed
 rpm -qa | grep -E "nfs-utils.*" || yum -y install nfs-utils
+
 service nfs restart
+if [ $? -ne 0 ]; then
+    service nfs-server restart
+fi
+
 mkdir /media/nfs-folder
 mkdir /nfs-mount-point
 echo "/media/nfs-folder     $ip*(rw,sync,no_root_squash)" > /etc/exports
 exportfs -a
+
 service nfs restart
+if [ $? -ne 0 ]; then
+    service nfs-server restart
+fi
+
 mount -o hard,nolock "$ip":/media/nfs-folder /nfs-mount-point/

